Examples/Pass/arc.cvf

```diff
@@ -44,16 +34,12 @@ method drop() {
       {| noCnt() |}
         <free = (true)>;
       {| emp |}
-    } else {
-      {| emp |}
-        dummy = dummy; // skip;
-      {| emp |}
     }
   {| emp |}
 }
 
 view error();
```

Comment:
I already went through the motions of doing it before we talked about this :(  so I've gone ahead and added this to the PRQ.  I think that's the last change I'm planning on this branch now—can this be merged?